testing/kuttl/e2e/exporter/13--check-exporter.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      set -e
+      PRIMARY=$(
+        kubectl get pod --namespace "${NAMESPACE}" \
+          --output name --selector '
+            postgres-operator.crunchydata.com/cluster=exporter-tls,
+            postgres-operator.crunchydata.com/role=master'
+      )
+
+      # Wait some time so that the instance pod is redeployed
+      # TODO: automate this wait
+      sleep 30
```

Comment:
🌱 Hmm. The status we have is the revision hash. My only idea atm is to wait for that to change. That requires storing the old value (before 12--disable-tls.yaml) somewhere (annotation?) then looping here with sleep until the status `!=` the old value.

Reply:
Would the revision hash help here? I need the instance pod to be rebuilt so that exporter is restarted without tls. Wouldn't the revision hash be updated when the StatefulSet is updated? Would that have a concept of if the pod has stopped/restarted?

Reply:
I spent a lot of time attempting to `kubectl wait` for the pod to be not ready and then ready again, but it got to the point where I just wanted/needed to be done with it. There is definitely room for improvement